CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/AnalyticsManager.swift

```diff
@@ -95,30 +95,15 @@ public class AnalyticsManager {
     static func track(event: AnalyticsEvent,
                       screenNameString: String? = nil,
                       properties: [AnalyticsProperty] = []) {
-        guard let mixpanelInstance, amplitudeInitialised else {
-            let queuedEvent = QueuedAnalyticsEvent(event: event,
-                                                   screenNameString: screenNameString,
-                                                   properties: properties)
-            eventsQueue.append(queuedEvent)
-            return
+        let queuedEvent = QueuedAnalyticsEvent(event: event,
+                                               screenNameString: screenNameString,
+                                               properties: properties)
+        eventsQueue.append(queuedEvent)
+
+        // Process the event queue if SDKs are initialized
+        if mixpanelInstance != nil && amplitudeInitialised {
+            processEventsQueue()
```

Comment:
[clarification] Do we still to call this method in `    private static func handleAnalyticsSDKsInit() {` ? I think is not neccessary.